### PR TITLE
Move CIFAR10 example and update CI tests

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Move CIFAR10 example under advanced
- Remove virtualenv folder in CIFAR10
- Move tensorboard-streaming example inside experiment_tracking
- Update CIFAR integration test

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [x] Documentation updated.

### CODE REVIEW

File: examples/advanced/cifar10/cifar10-real-world/README.md [hunk 1]
```diff
@@ -9,36 +9,30 @@ For more information on real-world FL see [here](https://nvflare.readthedocs.io/
...
 ```
 pip install --upgrade pip
-pip install -r ./virtualenv/min-requirements.txt
+pip install -r ./requirements.txt
 ```
+
 (optional) if you would like to plot the TensorBoard event files as shown below, please also install
 ```
-pip install -r ./virtualenv/plot-requirements.txt
+pip install -r ./figs/requirements.txt
```
[the remark this answers is not in the document]
Comment (holgerroth) [reply examples/advanced/cifar10/cifar10-real-world/README.md:22]: Yes. Let's only have one requirements.txt

File: examples/advanced/cifar10/cifar10-sim/README.md [hunk 2]
```diff
@@ -7,34 +7,27 @@ and [SCAFFOLD](https://arxiv.org/abs/1910.06378) algorithms using NVFlare's FL s
...
+> **_NOTE:_**  We recommend either using a containerized deployment or virtual environment, 
+> please refer to [getting started](https://nvflare.readthedocs.io/en/latest/getting_started.html).
+
 Set `PYTHONPATH` to include custom files of this example:
 ```
 export PYTHONPATH=${PWD}/..
 ```
 
-### 2. Download the CIFAR-10 dataset 
+## 2. Download the CIFAR-10 dataset
```
[the remark this answers is not in the document]
Comment (holgerroth) [reply examples/advanced/cifar10/cifar10-sim/README.md:30]: The intention was to keep these two examples standalone without the user having to go back and forth.

[the remark this answers is not in the document]
Comment (holgerroth) [reply examples/advanced/cifar10/cifar10-sim/README.md:30]: I was talking about the download instructions being both in the cifar10-sim and cifar10-read-world. That way, each can stand alone depending on what the user is interested in.

File: examples/advanced/cifar10/cifar10-real-world/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply examples/advanced/cifar10/cifar10-real-world/README.md:22]: sure will do

**Review by holgerroth — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**